Python-path-traversal-files-main/CJuanvip/CMEOptions/sm.py

```
@@ -0,0 +1,173 @@
+	dte = (settlements["options"][month]["expiration_date"] - settlements["settlement_date"]).days
+	change = daily_change(contract, delta_equivalent, dte)
+
+	average_options = wi.get_average_option(settlements["options"][month])
+	avg_call = average_options["CALL"]
+	avg_put = average_options["PUT"]
+	avg_option = average_options["TOTAL"]
+
+	price = settlements["options"][month]["underlying"]["price"]
+	und_price = sp.decimal_to_ticks(price, symbol)
+
+	if type(change) == type(""):
```

Ruff (0.13.3)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)